concepts/05 UI Components/List/00 Getting Started with List/07 Reorder Items.md

```diff
@@ -0,0 +1,78 @@
+The List supports drag & drop functionality that allows users to reorder list items. To configure it, define the [itemDragging](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDragging) object. Within this object, set the [allowReordering](/Documentation/ApiReference/UI_Components/dxSortable/Configuration/#allowReordering) property to **true**.
```

Comment:
I'm simplifying the first sentence. It's a compound sentence (two subject with corresponding verbs) and it's a pretty simple thought that doesn't seem to require it. What we want to say is that "List supports drag-and-drop item reordering". Maybe we want to avoid the "ing" word though. 

In the second sentence, I wouldn't use the pronoun (it). Use them when the noun their refer to is very close and clear. In this case it's closer to the beginning of the preceding sentence.

```suggestion
Users can drag and drop list items to reorder them. To configure this functionality, define the [itemDragging](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDragging) object. Within this object, set the [allowReordering](/Documentation/ApiReference/UI_Components/dxSortable/Configuration/#allowReordering) property to **true**.
```